apps/remote_control/lib/lexical/remote_control/code_intelligence/definition.ex

```diff
@@ -32,6 +34,28 @@ defmodule Lexical.RemoteControl.CodeIntelligence.Definition do
     {:ok, nil}
   end
 
+  defp maybe_move_cursor_to_next(type, document, line, column)
```

Comment:
naming: What does this function do? What is next?

Reply:
move the cursor from

```elixir
   |def func() do
```

to 

```elixir
  def |func() do
```

ElixirSense changed its default behavior.

Do you think `maybe_move_cursor_to_next_node` is better?

Reply:
Compared to using a zipper, do you think it would be more appropriate for us to define a `suffix_tokens` in `Lexical.Ast.Env` for cursor movement?